src/components/Breadcrumb.vue

```diff
@@ -62,7 +63,7 @@ export default {
           const { data } = await this.girderRest.get(`${type}/${_id}`);
           breadcrumb.unshift(this.extractCrumbData(data));
         }
-        if (this.rootLocation) {
+        if (rootLocationAllowed) {
```

Comment:
I think this is a matter of personal preference.  Inside computed properties, I like to put all "dependencies" at the top as documentation.  I find myself scanning computed functions for dependencies much more often than resolving local variables, particularly if I know it's already a prop, so that's the case this optimizes for.

That being said, I don't care that strongly.  Can revert if you'd like.

Reply:
It would be great if you would, be because a simple `this.rootLocationAllowed` is enough here.

Reply:
After changing this, I noticed that the reactivity system was no longer able to depend on rootLocationAllowed.  Checking and unchecking in the demo didn't work anymore.  Reverting fixed it.

I'll probably keep putting dependencies at the top like this in the future.  I don't know why AsyncComputed works differently for this case.

Reply:
Oh, make sense it doesn't work. All dependency of async computed needs to be accessed before the first async function. My request accidentally exactly broke it. 😂